client/src/components/Game/Timer.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { withFirebase } from '../Firebase';
+import { withRouter } from 'react-router';
+import { compose } from 'recompose';
+
+class Timer extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      minutes: 99,
+      seconds: 99,
```

Comment:
what is the reasoning behind initial values of 99?

Reply:
the initial state of the minutes will show up because it takes a half a second for the startGame timestamp to populate the variable. I am just using those as dummy values so that everything is defined

Reply:
Are you saying that 99 will show up on the screen?

Reply:
FYI [checking for existence](https://github.com/googleinterns/Pictophone/blob/1f80a4a21ddae937cb857549ecbf8cb5254164d3/client/src/components/Game/Endgame.js#L93) is a common loading practice in React. There are examples in [other parts](https://github.com/googleinterns/Pictophone/blob/1f80a4a21ddae937cb857549ecbf8cb5254164d3/client/src/components/Game/Canvas.js#L256) of this codebase too. In this case you can do `{this.state.seconds && this.state.minutes <component to be rendered>}` to avoid initializing to an arbitrary value

Reply:
it shows up for a split second while the numbers are rendering. i get a no-undef error when i try to leave the state fields undefined